setuptools/ssl_support.py

```diff
@@ -225,38 +224,14 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-@once
 def get_win_certfile():
-    try:
-        import wincertstore
-    except ImportError:
-        return None
-
-    class CertFile(wincertstore.CertFile):
-        def __init__(self):
-            super(CertFile, self).__init__()
-            atexit.register(self.close)
-
-        def close(self):
-            try:
-                super(CertFile, self).close()
-            except OSError:
-                pass
-
-    _wincerts = CertFile()
-    _wincerts.addstore('CA')
-    _wincerts.addstore('ROOT')
-    return _wincerts.name
```

Comment:
Don't drop this feature. Either replace it with https://github.com/python/cpython/blob/db3ff76da19004f266b62e98a81bdfd322861436/Lib/ssl.py#L555-L575, replace it with ``SSLContext.load_default_certs`` or even better: use ``ssl.create_default_context()``.

Reply:
so it turns out the code already does use `ssl.create_default_context(` where available. But now it's always available so the whole module can go!